pkg/authenticator/config/config.go

```diff
@@ -143,5 +149,17 @@ func populateConfig() (*Config, error) {
 		defaultConfig.ClientCertPath = envVal
 	}
 
+	// Parse client cert retry count limit if one is provided from env
+	defaultConfig.ClientCertRetryCountLimit = DefaultClientCertRetryCountLimit
+	clientCertRetryCountLimitString := os.Getenv("CONJUR_CLIENT_CERT_RETRY_COUNT_LIMIT")
+	if len(clientCertRetryCountLimitString) > 0 {
+		parsedClientCertRetryCountLimit, err := strconv.Atoi(clientCertRetryCountLimitString)
+		if err != nil {
+			return nil, log.RecordedError(log.CAKC034E, err.Error())
+		}
+
+		defaultConfig.ClientCertRetryCountLimit = parsedClientCertRetryCountLimit
+	}
```

Comment:
Not a blocker since it's old code but I think we've reached a point where some of this env var parsing could be done via helper methods

Reply:
thanks, leaving this out from this PR. i guess this helper should be used in secretless and secrets-provider as well.